### PR TITLE
Added landing page, button styles

### LINKED ISSUE #18: Fix the button design of landing page
Right now the button doesn't have any styling.
Design should be identical to that in https://github.com/UdacityMobileWebScholarship/udabot/blob/master/Guidelines/Screen1.jpg

This is the file where you need to look at for fixing this issue: https://github.com/UdacityMobileWebScholarship/udabot/tree/frontend/Frontend/udabot/src/componenets/CustomButton

### PR DESCRIPTION
**Pull request by blenderskool**
# Improved the landing page styles and added the button styles

Based on the design guidelines, styles for button have been added. The landing page is now responsive and matches with the design. Additionally, **'Open Sans' font had to be imported** since it wasn't being used.

Fixes #18 #19 (Both are a part of Landing page, hence submitting PR for both)

### CODE REVIEW

**Comment by blenderskool** [comment]
Also, Please note that Pixel units **don't support decimal places**, hence I have used **rem** units to make the layouts match visually similar to design

**Comment by Biboswan** [comment]
@blenderskool. Design seems good to me. I know react bootstrap was already added. But did it played any role here. I see without bootstrap also your design seems to work. Please remove it and check once. I mean changing `button:btn` to `button`.

**Comment by blenderskool** [comment]
@Biboswan I have checked if react bootstrap was being used, and I feel it isn't working properly. Removing my styles did not give the button default Bootstrap button styles. Not sure if it is supposed to work like this though :thinking: 

Changing `button.btn` to `button` also made no changes

**Comment by blenderskool** [comment]
Also keeping the styles to `button` ended up modifying the suggested message buttons in the chat interface, so I think we have to keep `button.btn`

**Comment by Biboswan** [comment]
have you replaced `<Button>` with `<button>` and checked

File: Frontend/udabot/src/componenets/CustomButton/CustomButton.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import {Button} from 'react-bootstrap';
```
Comment (Biboswan) [inline Frontend/udabot/src/componenets/CustomButton/CustomButton.tsx]: remove this

**Comment by Biboswan** [comment]
I meant making changes simultaneously.
replace `<Button>` with `<button>`
change occurrences of button.btn to button

**Comment by blenderskool** [comment]
Oh ok, I will remove the usage of Bootstrap button. Also, note that if I use `button` in CSS then it will be a problem in other layouts where buttons have different style

**Comment by Biboswan** [comment]
Only where I use CustomButton component this css will be used for that
particular button.

**Comment by blenderskool** [comment]
@Biboswan I have removed the unused Bootstrap button, it should probably be fine now :slightly_smiling_face:

**Comment by Biboswan** [comment]
thanls @blenderskool

**Pull request merged**